frappe/patches.txt

```diff
@@ -184,6 +184,7 @@ frappe.patches.v13_0.jinja_hook
 frappe.patches.v13_0.update_notification_channel_if_empty
 frappe.patches.v13_0.set_first_day_of_the_week
 execute:frappe.reload_doc('custom', 'doctype', 'custom_field')
+execute:frappe.reload_doc('desk', 'doctype', 'workspace_quick_list')
```

Comment:
There's separate patch for this `reload_workspace_child_tables` which probably needs to merged with original workspace patch.

@shariquerik ?